starcheck/calc_ccd_temps.py

```diff
@@ -198,9 +211,13 @@ def get_ccd_temps(oflsdir, outdir='out',
     # Get temperature telemetry for 1 day prior to min(last available telem,
     # backstop start, run_start_time) where run_start_time is for regression
     # testing.
-    tlm_end_time = min(fetch.get_time_range('aacccdpt', format='secs')[1],
-                       bs_start.secs, run_start_time.secs)
-    tlm = get_telem_values(tlm_end_time, ['aacccdpt'], days=1)
+    if use_maude:
+        tlm_end_time = min(bs_start.secs, run_start_time.secs)
```

Comment:
This assumes that MAUDE always has some telemetry within 1.0 day of when this is being run (for the usual case of reviewing loads in the future). That can fail if we miss passes for a day. 

Since the MAUDE option is not the default this is OK, but worth noting that the end time of telemetry in MAUDE can be easily obtained with a direct `maude` query without a start/stop time:
```
import maude
dat = maude.get_msids("aacccdpt")  # returns the latest sample
maude_end_time = dat["data"][0]["times"][0]
```

Reply:
Good catch.